src/muiThemeable.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
 export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+  function MuiComponent(props, {muiTheme = getMuiTheme()}) {
```

Comment:
@oliviertassinari Actually, one more thought on function vs class. @neverfox brought up that there are still some kinks being worked out as it relates to hot-reloading for components that are defined as functions. So another reason to switch to a class would be if we're experiencing issues with that. I think you mentioned that the catch errors broke with components decorated with `muiTheme`? I'd be curious if a switch to class would fix that.

That all being said, I'm pretty sure the last PR/branch you were testing wasn't implemented with a function but was implemented with `React.createClass`. Could be wrong though...